src/public/bulkUpload.csv

```diff
@@ -0,0 +1,3 @@
r of Terms Permitted,Organization Governing Body Compensation,Organization Governing Body Financial Oversight,Organization Board Chair,Organization Board Chair Title,Organization Board Members,Organization Board Women,Organization Board Men,Organization Board Intersex,Organization Board African Descent,Organization Board Asian Descent,Organization Board European Descent,Organization Board Indigenous Peoples,Organization Board Latin American Descent,Organization Board Middle Eastern Descent,Organization Board Multiracial Descent,Organization Paid Staff,Organization Senior Staff Women,Organization Senior Staff Men - Men,Organization Senior Staff Intersex People,Organization Senior Staff African Descent,Organization Senior Staff Asian Descent,Organization Senior Staff European Descent,Organization Senior Staff Indigenous Peoples,Organization Senior Staff Latin American Descent,Organization Senior Staff Middle Eastern Descent,Organization Senior Staff Multiracial Descent,Organization Conflict of Interest Policy,Proposal Date,Proposal Name,Proposal Title,Proposal Type,Proposal Focus,Proposal Summary,Proposal Primary Contact Name,Proposal Primary Contact Title,Proposal Primary Contact Phone,Proposal Primary Contact Email,Proposal Submitter Prefix,Proposal Submitter First Name,Proposal Submitter Middle Initial,Proposal Submitter Last Name,Proposal Submitter Suffix,Proposal Submitter Title,Proposal Submitter Phone,Proposal Submitter Extension,Proposal Submitter Mobile Phone,Proposal Submitter Email,Proposal Submitter Street Address 1,Proposal Submitter Street Address 2,Proposal Submitter City/Town,Proposal Submitter State/Province,Proposal Submitter State Code,Proposal Submitter Region,Proposal Submitter Zip Code,Proposal Submitter Name,Proposal Amount Requested,Proposal Budget,Proposal Budget Narrative,Proposal Project Budget,Proposal Fiscal Sponsor?,Proposal Fiscal Sponsor Information,Proposal Fiscal Sponsor Name,Proposal Fiscal Sponsor EIN,Proposal Fiscal Sponsor Contact First Name,Proposal Fiscal Sponsor Cantact Last Name,Proposal Fiscal Sponsor Contact Email,Proposal Regrant,Proposal Regrant Subgrantees,Proposal Subgrantee List,Proposal Expenditure Responsibility,Proposal Equivalency Determination,Proposal Tipping Amount,Proposal Funding Summary,Proposal Start Date,Proposal End Date,Proposal Duration,Proposal Location of Work,Proposal Age Group,Proposal Ethnicity,Proposal Need,Proposal Activities,Proposal Tactics and Methods,Proposal Strategy,Proposal Organizational Strengths,Proposal Equity,Proposal Engagement,Proposal Challenges and Risks,Proposal Related Programs,Proposal Supplemental Information,Proposal Results,Proposal Project Outcomes,Proposal Anticipated Impact,Proposal Sustainability,Proposal IRB Approval Needed,Proposal IRB Approval Obtained,Organization Lobbying Activities,Proposal Legislation,Proposal Lobbying more than Grant,PDC Share Data,Fundraising Expense Ratio,Program Expense Ratio,Administrative Expense Ratio,Working Capital Ratio
```

Comment:
I'm hesitant about including this. I see the UX value, but it feels potentially problematic. How will you reliably distinguish a row of labels from a row of content?

Reply:
Happy to take this out as well -- the idea here is that the first two (well, three if we include instructions) would always be skipped.  Content would start on row 4.

Reply:
IMO we should count on an entirely different interface for humanizing field keys than the CSV template. Here, the only function of the label row is explanatory; which is great, but immediately complicates the story. I imagine the user thinking: can I remove this row if I want? can I customize the labels by tweaking the wording of these fields?

I think it's best to just provide the key row. That's a much more stable rule to enforce. "First row is base field keys. Bam."

Reply:
> Another benefit of taking this out is that suddenly our "template" is actually just a "very easy to construct yourself, if you want" specification.

Agreed!